parlai/core/torch_generator_agent.py

```diff
@@ -905,6 +904,11 @@ def eval_step(self, batch):
                     except KeyError:
                         logging.error("Decoding error: %s", tokens)
                         continue
+            # reorder slightly so match_batch() handles correctly
+            stride = int(len(beam_texts) / batch.batchsize)
```

Comment:
`len(beam_texts) // batch.batchsize` python3 is shorthand

Reply:
:o